deployment/roles/monitor/tasks/jumpbox.yml

```diff
@@ -0,0 +1,12 @@
+- name: Launch container
+  shell: docker-compose up -d
+  args:
+    chdir: "{{ bridge_path }}/monitor"
+- name: Initial checks 1/2
+  shell: docker-compose exec -T monitor /bin/bash -c "timeout -s 9 5m node checkWorker.js"
```

Comment:
Should this command (and the one below) be run once during initial launch? It doesn't look like they worked in my test because no `responses/*.json` files were created in container's filesystem
```
~$ sudo docker exec -it monitor_monitor_1 /bin/bash
root@0c883ae1a263:/mono/monitor# ls
alerts.js	checkWorker2.js  crontab.example  getBalances.js	 index.js   package.json  stuckTransfers.js  utils
checkWorker.js	checkWorker3.js  eventsStats.js   getShortEventStats.js  logger.js  responses	  test-srv.js	     validators.js
root@0c883ae1a263:/mono/monitor# cd responses/
root@0c883ae1a263:/mono/monitor/responses# ls
# empty list
```

Reply:
And later if I run workers manually, these files are created
```
root@0c883ae1a263:/mono/monitor# node checkWorker
checkWorker.js   checkWorker2.js  checkWorker3.js  
root@0c883ae1a263:/mono/monitor# node checkWorker.js 
2019-07-08T19:57:09.441Z (+0ms) [validators] Fetched gasprice: 1
root@0c883ae1a263:/mono/monitor# node checkWorker2.js 
root@0c883ae1a263:/mono/monitor# ls responses/
alerts.json  eventsStats.json  getBalances.json  validators.json
```